### PR TITLE
Removes the extra L from tequila.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Spellchecking and bugfixing.

## Why It's Good For The Game
Fixes Tequila bottles starting empty.

## Changelog
:cl:
fix: Tequila bottles now come with actual tequila inside.
spellcheck: Removed the extra L from many istances of "tequila".
/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/food/drinks/bottle.dm [hunk 1]
```diff
@@ -116,7 +116,7 @@
 	list_reagents = list("vodka" = 100)
 
 /obj/item/reagent_container/food/drinks/bottle/tequilla
-	name = "\improper Caccavo Guaranteed Quality Tequilla"
+	name = "\improper Caccavo Guaranteed Quality Tequila"
 	desc = "Made from premium petroleum distillates, pure thalidomide and other fine quality ingredients!"
 	icon_state = "tequillabottle"
```
Comment (ShizCalev) [inline code/game/objects/items/reagent_containers/food/drinks/bottle.dm]: tequilla

File: code/game/objects/items/reagent_containers/food/drinks/bottle.dm [hunk 2]
```diff
@@ -115,10 +115,10 @@
 	center_of_mass = list("x"=17, "y"=3)
 	list_reagents = list("vodka" = 100)
 
-/obj/item/reagent_container/food/drinks/bottle/tequilla
-	name = "\improper Caccavo Guaranteed Quality Tequilla"
+/obj/item/reagent_container/food/drinks/bottle/tequila
+	name = "\improper Caccavo Guaranteed Quality Tequila"
 	desc = "Made from premium petroleum distillates, pure thalidomide and other fine quality ingredients!"
-	icon_state = "tequillabottle"
+	icon_state = "tequilabottle"
```
Comment (DominikPanic) [inline code/game/objects/items/reagent_containers/food/drinks/bottle.dm]: You're changing icon states, are you sure these exist in the .dmi?

File: code/game/objects/items/reagent_containers/food/drinks/drinkingglass.dm [hunk 3]
```diff
@@ -192,11 +192,11 @@
...
 					icon_state = "bravebullglass"
 					name = "Brave Bull"
-					desc = "Tequilla and Coffee liquor, brought together in a mouthwatering mixture. Drink up."
+					desc = "Tequila and Coffee liquor, brought together in a mouthwatering mixture. Drink up."
 					center_of_mass = list("x"=15, "y"=8)
-				if("tequillasunrise")
-					icon_state = "tequillasunriseglass"
-					name = "Tequilla Sunrise"
+				if("tequilasunrise")
+					icon_state = "tequilasunriseglass"
```
Comment (DominikPanic) [inline code/game/objects/items/reagent_containers/food/drinks/drinkingglass.dm]: Same here

File: code/game/objects/items/reagent_containers/food/drinks/drinkingglass.dm [hunk 4]
```diff
@@ -134,9 +134,9 @@
 					name = "Glass of Vermouth"
 					desc = "You wonder why you're even drinking this straight."
 					center_of_mass = list("x"=16, "y"=12)
-				if("tequilla")
-					icon_state = "tequillaglass"
-					name = "Glass of Tequilla"
+				if("tequila")
+					icon_state = "tequilaglass"
```
Comment (DominikPanic) [inline code/game/objects/items/reagent_containers/food/drinks/drinkingglass.dm]: Same here.

File: code/game/objects/items/reagent_containers/food/drinks/bottle.dm (hunk 2 above)
Comment (Ghommie) [reply code/game/objects/items/reagent_containers/food/drinks/bottle.dm]: I accordingly changed the icon_state names in the .dmi too.

**Pull request merged**